accord-core/src/main/java/accord/impl/progresslog/DefaultProgressLogs.java

```diff
@@ -54,7 +56,7 @@ static boolean pauseForTest(DefaultProgressLog progressLog)
             return false;
 
         if (!paused.containsKey(progressLog))
-            paused.putIfAbsent(progressLog, () -> progressLog.commandStore.execute(progressLog));
+            paused.putIfAbsent(progressLog, () -> progressLog.commandStore.execute(empty(), progressLog));
```

Comment:
discussed offline: looks like we may want to add `begin` here, since when runnable runs, we only create an async chain but do not start it.